jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -106,8 +106,13 @@ def buildExclusions = [
   [ /sharedlibs_shared/,              anyType,     lt(9)   ],
 
   // OSX ---------------------------------------------------
+  [ /osx11-release-pkg/,              releaseType, lte(15) ],
+  [ /osx11-release-tar/,              releaseType, lte(15) ],
+  [ /osx1015-release-pkg/,            releaseType, lt(15)  ],
```

Comment:
```suggestion
  [ /osx1015-release-pkg/,            releaseType, gt(15)  ],
```

Reply:
Actually perhaps for consistency with the rest of the file we should use `lt(16)` for the above lines and `gte(16)` for this one?